config/config.sample.yaml

```diff
@@ -8,26 +8,26 @@ app_auth: #Get these from twitter by creating an application through the API sit
   consumer_secret: BLAHBLAHBLAH
 
 hashtags:
-  enable: true
-  admin_only: false
+  enable: true # Enable processing hashtag rooms and creating new ones
+  admin_only: false # NOT USED. Only an admin listed can create these rooms.
```

Comment:
I had previously set up a system to only allow listed users the ability to bridge and to control various bits of the bridge. I will eventually be making use of them but I'll probably comment them out for now to avoid confusion.